frontend/src/features/projects/dispose/ProjectDisposeLayout.tsx

```diff
@@ -76,6 +121,7 @@ const ProjectDisposeLayout = ({ match, location }: { match: Match; location: Loc
           />
           {getProjectRequest?.isFetching !== true ? (
             <Container fluid className="step-content">
+              {preDraft ? <ProjectDraftStep formikRef={formikRef} /> : null}
```

Comment:
before the first step it will load the basic form that it will use to create the project in the draft step